app/src/main/java/ch/hikemate/app/model/route/HikesViewModel.kt

```diff
@@ -478,7 +568,7 @@ class HikesViewModel(
    *
    * @return True if the operation is successful, false otherwise.
    */
-  private suspend fun refreshSavedHikesCacheAsync(): Boolean =
+  private suspend fun refreshSavedHikesCacheAsync(forceOverwriteHikesList: Boolean): Boolean =
```

Comment:
Suggestion: Maybe here you could add why exactly the `forceOverwriteHikesList` parameter is used for, since it isn't really clear.

Reply:
I added some docstring to help 👌🏻